src/Commands/User/UserCommandGroup.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Ipfs.CoreApi;
+using Org.BouncyCastle.Ocsp;
+using OwlCore.Diagnostics;
+using OwlCore.Kubo;
+using OwlCore.Nomad.Kubo;
+using OwlCore.Nomad.Kubo.Events;
+using OwlCore.Storage;
+using Remora.Commands.Attributes;
+using Remora.Discord.API.Abstractions.Rest;
+using Remora.Discord.Commands.Contexts;
+using Remora.Discord.Commands.Extensions;
+using Remora.Discord.Commands.Feedback.Services;
+using Remora.Results;
+using WindowsAppCommunity.Discord.ServerCompanion.Services;
+using WindowsAppCommunity.Sdk.Nomad;
+using static Org.BouncyCastle.Math.EC.ECCurve;
+using WindowsAppCommunity.Discord.ServerCompanion.Extensions;
+
+namespace WindowsAppCommunity.Discord.ServerCompanion.Commands.User
+{
+    public class UserCommandGroup(INomadRepoService nomadRepoService, IInteractionContext interactionContext, IFeedbackService feedbackService, IDiscordRestInteractionAPI interactionAPI, IDiscordRestChannelAPI channelApi, IDiscordRestGuildAPI guildApi, ICommandContext context) : Remora.Commands.Groups.CommandGroup
+    {
+        [Command("createUser")]
```

Comment:
Command groups should follow this structure, as shown to the end user:
```
waccli <entityType> <memberName> <operation> <params/values>
```

This should be on all three root entity types, not just this User file.